bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1541,8 +1541,22 @@ use_crop=".true.">lnd/clm2/surfdata_map/ctsm5.1.dev052/landuse.timeseries_mpasa1
 
 <!-- SNICAR (SNow, ICe, and Aerosol Radiative model) datasets -->
 <!--  ***********  Resolution independent:   ***********  -->
-<fsnowoptics >lnd/clm2/snicardata/snicar_optics_5bnd_c090915.nc</fsnowoptics>
-<fsnowaging  >lnd/clm2/snicardata/snicar_drdt_bst_fit_60_c070416.nc</fsnowaging>
+<fsnowoptics480>lnd/clm2/snicardata/snicar_optics_480bnd_c012422.nc</fsnowoptics480>
```

Comment:
@cenlinhe new question:
When the code runs with 5 bands, does it really need to know about the 480bnd file? (And vice versa.) If not, then I can go ahead and clarify that in the code.

Reply:
no, it does not. if running with 5 bands, only the 5-band file is needed. vice versa.